server/template/upload_logs.xhtml

```diff
@@ -0,0 +1,58 @@
+<html py:extends="master.xhtml" py:strip="True">
+
+<head py:block="head" py:strip="True">
+  <title py:block="master_title">Upload Logs</title>
+</head>
+
+<body py:block="body" py:strip="True">
+  <div class="row">
+    <div class="col-md-12">
+      <form name="upload_form" action="upload_logs" method="post" enctype="multipart/form-data">
+        <div class="form-group" id="message">
+          <textarea name="message" rows="10" cols="60" placeholder="Enter logs here or upload a text file below" >${raw_logs}</textarea>
+        </div>
+        <div class="form-group" id="message">
```

Comment:
So I added those when I was messing around with trying to get it to be required in the form to have either a file uploaded or text pasted into the textbox, before hitting submit